File: Day29/floydTriangle.js
function getPrintFloydTriangle(rows){
    let num = 1;
    for(let i = 1; i<= rows; i++){
        let row = ' '
        for(let j = 1; j <= i; j++){
            row = row + num + ' ';
            num++;
        }
        console.log(row);
    }
}
let rows = parseInt(prompt("Enter the number of rows :  "));

getPrintFloydTriangle(rows);

File: Day29/floyd_triangle.py
"""Function to print Floyd triangle pattern"""
def get_print_floyd_triangle(rows):
    """Initialize a variable to keep track of the current number."""
    num = 1
    #Outer loop to iterate over the rows
    for i in range(rows):
        #inner loop to print number in each rows
        for _ in range(i+1):
            #print the current number and increment it.
            print(num , end= " ")
            num +=  1
        print()

input_row = int(input("Enter the number of rows :  " ))
get_print_floyd_triangle(input_row)
